modules/debts_api/lib/debts_api/v0/financial_status_report_service.rb

```diff
@@ -181,12 +180,21 @@ def send_vha_confirmation_email(_status, options)
     private
 
     def add_vha_specific_data(form_submission)
+      combined_adjustments(form_submission)
       form = form_submission.form
       form['transactionId'] = form_submission.id
       form['timestamp'] = form_submission.created_at.strftime('%Y%m%dT%H%M%S')
       streamline_adjustments(form)
     end
 
+    def combined_adjustments(submission)
+      form = submission.form
+      if form.public_metadata['combined']
```

Comment:
Think this should be reading the public metadata from the submission object instead of the form

```suggestion
      if submission.public_metadata['combined']
```

Reply:
good catch